WhichSpace/AppDelegate.swift

```diff
@@ -9,13 +9,14 @@
 import Cocoa
 
 @NSApplicationMain
-class AppDelegate: NSObject, NSApplicationDelegate {
+class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
 
     @IBOutlet weak var window: NSWindow!
     @IBOutlet weak var statusMenu: NSMenu!
 
     var icons = [NSImage]()
-    let statusBarItem = NSStatusBar.systemStatusBar().statusItemWithLength(-1)
+    let statusBarItem = NSStatusBar.systemStatusBar().statusItemWithLength(27)
```

Comment:
That's the width of the status item, which I wanted to specify as I was drawing it myself